### PR TITLE
Feature/doc improvements

### PR DESCRIPTION
**Pull request by mprorock**
prettied up some of the html generation, getting ready to overhaul some of the core docs for both developer and public facing - @OR13 @Therecanbeonlyone1969 @mkhraisha let me know if there are key areas you want in or out of docs

Things that are going to get in here:

- [x] format outbound html after generation
- [x] make sure readme in main schemas covers build process adequately
- [x] make sure top level readme covers goals, getting started, how to add a type, and build process
- [x] cover env variables related to build output useful for debugging
- [x] main spec to cover SMEs and Contacts for types of data
- [x] code best practices (e.g. not using dynamic dates, naming, inheritance, and expected level of denormalization)
- [x] some missing things that need help, e.g. allOf on the schema side
- [ ] basic flow diagram of vocab build process for a given element

### CODE REVIEW

**Comment by mprorock** [comment]
will add some nice diagrams later in another pr

File: README.md [hunk 1]
```diff
@@ -1,14 +1,79 @@
...
 
 ![CI](https://github.com/w3c-ccg/traceability-vocab/workflows/CI/badge.svg) [![CD](https://github.com/w3c-ccg/traceability-vocab/actions/workflows/cd.yml/badge.svg)](https://github.com/w3c-ccg/traceability-vocab/actions/workflows/cd.yml)
 
 This specification describes a Linked Data vocabulary for asserting Verifiable Credentials related to traceability information, such as chemical properties, mechanical properties, country of origin, and other attributes used to determine the status of a products and materials in a supply chain.
 
 We encourage contributions meeting the [Contribution Guidelines](CONTRIBUTING.md). While we prefer the creation of issues and Pull Requests in the GitHub repository, discussions often occur on the [public-credentials](http://lists.w3.org/Archives/Public/public-credentials/) mailing list as well.
 
+## Getting Started
+
+If youare just interested in the vocabulary itself, the items contained within, and the example Verifiable Credentials, plase see the [spec itself](https://w3c-ccg.github.io/traceability-vocab/).
```
Comment (TallTed) [inline README.md:11]:
```suggestion
If you are only interested in the vocabulary itself, the items contained therein, and/or the example Verifiable Credentials, please see the [spec itself](https://w3c-ccg.github.io/traceability-vocab/).
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,79 @@
...
+
+If you are a developer who is interested in working directly with the vocabulary, possibly add schemas, as well as to run tests and view the details of how the JSON-JD and resultant Verifiable Credentials are build, then read on.
```
Comment (TallTed) [inline README.md:13]:
```suggestion
If you are a developer who is interested in working directly with the vocabulary, possibly adding schemas, and/or running tests and viewing details of how the JSON-JD and resultant Verifiable Credentials are built, then read on.
```

File: README.md [hunk 3]
```diff
@@ -1,14 +1,79 @@
...
+
+### Developer Setup
+
+To get started with the repository you will need at a minimum [Node.js](https://nodejs.org/en/) and [git](https://git-scm.com/) in order to checkout the project, fetch other dependencies, and build and run the project.
```
Comment (TallTed) [inline README.md:17]:
```suggestion
To get started with the repository -- to check-out the project, fetch other dependencies, and build and run the project -- at a minimum, you will need [`Node.js`](https://nodejs.org/en/) and [`git`](https://git-scm.com/).
```

Comment (mprorock) [inline README.md:17]:
```suggestion
To get started building, testing, or contributing the repository you will need at a minimum [Node.js](https://nodejs.org/en/) and [git](https://git-scm.com/)
```

Comment (mprorock) [reply README.md:17]: putting an alternate wording in

File: README.md [hunk 4]
```diff
@@ -1,14 +1,79 @@
...
+
+Any changes that potentially introduce breaking or non-backwards compatible functionality MUST have a corresponing issue and discussion, and will require consensus from the editors in order to be introduced or to have any related Pull Requests accepted and merged.
+
+A one week (7 day) period will be provided for review of pull requests related to data schemas or project functionality prior to merge to allow suffient review time. Execptions may be made for essential documentation, or to allow for immediate hotfix of security issues or functionality breaking items.
+
+
 ## Ontology Structure
 
-This repository hosts both [json-schema](https://json-schema.org/) and [jsonld](https://json-ld.org/).
+This repository hosts both [json-schema](https://json-schema.org/) which it uses to create [jsonld](https://json-ld.org/).
```
Comment (mprorock) [inline README.md:76]:
```suggestion
This repository hosts [json-schema](https://json-schema.org/) which it uses to create [jsonld](https://json-ld.org/).
```

**Review by OR13 — APPROVED** [review]

**Pull request merged**